CocoaMarkdown/CMAttributedStringRenderer.m

```diff
@@ -20,6 +20,7 @@
 #import "Ono.h"
 
 @interface CMAttributedStringRenderer () <CMParserDelegate>
+@property (nonatomic, strong) NSArray<CMPlainTextTransformer *> *plainTextTransformers;
```

Comment:
Propose to make NSDictionary<CMNodeKey *, NSArray< CMPlainTextTransformer  *>> plainTextTransformersMap. Then as key we can create CMNodeKey object (nodeType, headerLevel).